### PR TITLE
End user installation instructions

### LINKED ISSUE #94: Write end user installation instructions
Current installation instructions in [README](https://github.com/informalsystems/atomkraft/blob/e28caa940deccabab7e6e37b7f2ca20483cae128/README.md) are for developers; we need to write the installation instructions for an end user, which would explain in details what needs to be installed, and how it's possible to do it. In particular, the following needs to be explained:

- Python 3.10 as a prerequisite for Atomkraft
- Atomkraft installation itself
- Possibilities of getting blockchain binaries, if they are not installed
- Jave RE, as a prerequisite for Apalache, if the user wants to work with models.

### PR DESCRIPTION
**Pull request by andrey-kuprianov**
[INSTALLATION.md](https://github.com/informalsystems/atomkraft/blob/andrey/install/INSTALLATION.md)

Closes #94.

### CODE REVIEW

**Comment by ivan-gavran** [comment]
I pushed [a commit](https://github.com/informalsystems/atomkraft/pull/96/commits/9286d3d21842cff36c987f3963705db0ba9568d6) with my suggestions.

What I tried to accomplish with the changes is the following:
 - give a solution to people who are looking for a quick c/p  (I added a section in the end and linked it from the beginning)
 - give a solution for obtaining python 3.10 without having to go through those lengthy `realpython.com` installation instructions. 

Let me know how it looks to you, @andrey-kuprianov . (It is quite an intervention to the original text so feel free to modify further or insist on reverting it if it looks like a completely wrong direction.)

**Comment by andrey-kuprianov** [comment]
Wow, it looks terrific, @ivan-gavran! Thanks for the additions:)

**Comment by andrey-kuprianov** [comment]
@ivan-gavran please ho ahead, review, and merge. From my side all good

**Comment by ivan-gavran** [comment]
Before merging, I'd like to get feedback from @rnbguy , especially for the TL;DR section: does this go through on Linux? In particular, is it true that `pyenv` is not needed there and that `python 3.10` will be installed when `apt` is updated?

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+your mileage may vary based on how much of that software you already have installed.
+In the following sections we describe how to install and what features are supported by different dependencies.
+If you are looking for a copy&paste solution to get Atomkraft up and running as quickly as possible,
+check the [TL;DR section](#tldr).
+
+## Installing Atomkraft
+
+Installing Atomkraft requires [Python](https://www.python.org) 3.10.
+
+If you don't have Python 3.10 already installed, please consult the [installation instructions](https://realpython.com/installing-python/) for different installation options.
```
Comment (rnbguy) [inline INSTALLATION.md:14]: Why do we reference to `realpython` blog here? Why not official instructions at https://www.python.org/downloads?

File: INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+These two commands set the `python` version to `3.10.5`.
+If you want to go back to your old Python version, run `pyenv global system`.
+
+Once Python 3.10 installed, install Atomkraft with `pip install atomkraft`.
+
+That's it! Please verify that the tool is working by issuing `atomkraft` on the command line.
+You should see something like this:
+
+![Atomkraft CLI](docs/images/cli.png)
```
Comment (rnbguy) [inline INSTALLATION.md:30]: It is a console output. why not just copy-paste the text from console in README?

**Review by rnbguy — COMMENTED** [review]
I added some comments. Otherwise, thanks for the write up :slightly_smiling_face:

**Pull request merged**